.github/workflows/bump-version.yml

```
@@ -0,0 +1,20 @@
+name: 'Bump Version'
+
+on:
+  workflow_dispatch:
+
+jobs:
+  version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Shall we use the latest one?

```suggestion
      - uses: actions/checkout@v3
```